templates/modelGeneric.hbs

```diff
@@ -18,5 +23,6 @@ export interface {{className name}}{{#if parents}} extends {{#each parents}}{{>f
 	{{#if readOnly}}readonly {{/if}}{{{quoteInvalidIdentifier serializedName}}}{{#unless required}}?{{/unless}}: {{{nativeType.serializedType}}};
 {{/each}}
 }
+{{/if}}
 {{/ifvex}}
 {{>modelNestedModels}}
```

Comment:
Perhaps this is the point where we check if the schema is multipart and _don't_ do the nested models, rather than having the multipart checks in the nested schema files. Or we might not actually need to _do anything_ as a multipart schema might not have any nested schemas? Or does it have the ones the generator makes for it? I'm curious about whether we can extend or change the auto generated schemas so we _can_ use them... as I feel like we might be losing type safety currently?